templates/types/streaming/nextjs/app/components/ui/chat/chat-message/chat-agent-events.tsx

```diff
@@ -20,10 +21,18 @@ const AgentIcons: Record<string, LucideIcon> = {
   publisher: icons.BookCheck,
 };
 
+type ProgressData = {
+  progress_id: string;
+  total_steps: number;
+  current_step: number;
+  step_msg: string;
```

Comment:
snake_case is weird in TS, how about avoiding it?
```suggestion
  id: string;
  total: number;
  current: number;
  msg: string;
```